relayer/app/app.go

```diff
@@ -1,4 +1,4 @@
-package relayer
```

Comment:
this was needed because generator will define the package as app (correctly) also I think it's better to have package named as they should be